facebook/delphiFacebook/R/contingency_aggregate.R

```diff
@@ -0,0 +1,361 @@
+    dfs_out[[aggregation]] <- bind_rows( lapply(dfs, function(groupby_levels) { 
+      groupby_levels[[aggregation]] 
+    }))
+  }
+  
+  ## Do post-processing.
+  for (row in seq_len(nrow(aggregations))) {
+    aggregation <- aggregations$id[row]
+    groupby_vars <- aggregations$group_by[[row]]
+    post_fn <- aggregations$post_fn[[row]]
+    
+    dfs_out[[aggregation]] <- dfs_out[[aggregation]][
+      rowSums(is.na(dfs_out[[aggregation]][, c("val", "sample_size", groupby_vars)])) == 0,
+    ]
+    
+    if (geo_level == "county") {
+      df_megacounties <- megacounty(dfs_out[[aggregation]], params$num_filter, groupby_vars)
+      dfs_out[[aggregation]] <- bind_rows(dfs_out[[aggregation]], df_megacounties)
+    }
+    
+    dfs_out[[aggregation]] <- apply_privacy_censoring(dfs_out[[aggregation]], params)
+    
+    ## *After* gluing together megacounties, apply the post-function
+    dfs_out[[aggregation]] <- post_fn(dfs_out[[aggregation]])
+  }
+  
+  return(dfs_out)
+}
+
+#' Produce estimates for all indicators in a specific target group.
+#' 
+#' @param group_df Data frame containing all data needed to estimate one group.
+#'   Estimates for `target_group` will be based on all of this data.
+#' @param aggregations Aggregations to report. See `aggregate_aggs()`.
+#' @param target_group A `data.table` with one row specifying the grouping
+#'   variable values used to select this group.
+#' @param geo_level a string of the current geo level, such as county or state, 
+#'   being used
+#' @param params Named list of configuration options.
+#' 
+#' @importFrom tibble add_column as_tibble
+#' @importFrom dplyr %>%
+#' 
+#' @export
+summarize_aggregations_group <- function(group_df, aggregations, target_group, geo_level, params) {
+  ## Prepare outputs.
+  dfs_out <- list()
+  for (row in seq_along(aggregations$id)) {
+    aggregation <- aggregations$id[row]
+    
+    dfs_out[[aggregation]] <- target_group %>%
+      as.list %>%
+      as_tibble %>%
+      add_column(val=NA_real_) %>%
+      add_column(se=NA_real_) %>%
+      add_column(sample_size=NA_real_) %>%
+      add_column(effective_sample_size=NA_real_) %>%
+      add_column(represented=NA_real_)
+  }
+  
+  for (row in seq_along(aggregations$id)) {
+    aggregation <- aggregations$id[row]
+    metric <- aggregations$metric[row]
+    var_weight <- aggregations$var_weight[row]
+    compute_fn <- aggregations$compute_fn[[row]]
+    
+    agg_df <- group_df[!is.na(group_df[[var_weight]]) & !is.na(group_df[[metric]]), ]
+    
+    if (nrow(agg_df) > 0)
+    {
+      s_mix_coef <- params$s_mix_coef
+      mixing <- mix_weights(agg_df[[var_weight]] * agg_df$weight_in_location,
+                            s_mix_coef, params$s_weight)
+      
+      sample_size <- sum(agg_df$weight_in_location)
+      total_represented <- sum(agg_df[[var_weight]] * agg_df$weight_in_location)
+      
+      ## TODO Fix this when same bit is fixed in `aggregate.R`, ~line 181
```

Comment:
Is there a git issue for tracking this fix yet? If so, cite the issue number here. If not, create one and then cite it in both places.